### ISSUE TITLE
Add custom format options

### ISSUE THREAD

**Issue by aaccomazzi** [adsabs/export_service#97]
We should support ADS classic's support for the following custom format directives:
```
* %ZAuthorSep:STRING - use STRING to separate author names
* %ZEOL:STRING - use STRING as end of record separator
```
For backwards compatibility we should recognize the following to directives as being equivalent to `%ZEncoding:Unicode`:
```
* %ZEncoding:UTF-8
* %ZEncoding:utf-8
```

**Comment by golnazads** [comment]
have been supporting both %ZEncoding:unicode and %ZEncoding:UTF-8
now any form of `unicode` and `UTF-8` (uppercase/lowercase) shall be supported

**Comment by aaccomazzi** [comment]
I found just two cases of `%ZAuthorSep` in the Classic user database.  Here is an example:
```
%ZAuthorSep:"; " \\bibitem%{R} %\10.10G %\t.\ {\\em %\j}\ {\\bf %\Y}, {\\em %\V}, %\p%-\P\. [%\d] \n
```

**Comment by golnazads** [comment]
implemented in v2.0.39.
